libmscore/sym.cpp

```diff
@@ -5321,6 +5321,7 @@ QVector<oldName> oldNames = {
       {"32' rest",                              SymId::rest32nd },            // rests.5
       {"64' rest",                              SymId::rest64th },            // rests.6
       {"128' rest",                             SymId::rest128th },           // rests.7
+      {"256' rest",                             SymId::rest256th },           // rests.8
```

Comment:
Where's 512th and 1024th rest? Probably not needed as they can't happen in 1.x score, so this change doesn't seem to be needed at all.
 As we recently dropped 1.x score support for 3.0, we may even get rid of the entire oldName stuff. Unless there are plans to have this PR getting integrated into the 2.0.4 branch too.